mlflow_databricks_artifacts/store/artifact_repo.py

```diff
@@ -0,0 +1,323 @@
+import base64
+import logging
+import os
+import posixpath
+import requests
+import uuid
+
+from mlflow import get_tracking_uri
+from mlflow.entities import FileInfo
+from mlflow.exceptions import MlflowException
+from mlflow.store import artifact_repo
+from mlflow.protos.service_pb2 import MlflowService, GetRun, ListArtifacts
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, INTERNAL_ERROR
```

Comment:
Rather than redefining MLflow service protos in this library, we attempt to import them from MLflow. The reason is that re-including them produces conflicts that are challenging to resolve (proto is not very tolerant of importing proto services / files with the same namespaces).